bench/Chainweb/Pact/Backend/ForkingBench.hs

```diff
@@ -186,7 +186,7 @@ mineBlock
     -> IO (T3 ParentHeader BlockHeader PayloadWithOutputs)
 mineBlock parent nonce pdb bhdb pact = do
     !r@(T3 _ newHeader payload) <- createBlock True parent nonce pact
-    addNewPayload pdb payload
+    addNewPayload pdb (succ $ _blockHeight $ _parentHeader parent) payload
```

Comment:
This is a hair split, but using `$` here isn't too helpful is it? I'd prefer just regular parens for argument passing to pure functions. `succ (_blockHeight (_parentHeader parent))` uses exactly the same number of characters as the dollar version, but is instantly friendly to readers of any other programming language.